lib/model/Schedule.dart

```diff
@@ -1,9 +1,7 @@
-import 'package:cloud_firestore/cloud_firestore.dart';
-
 class Schedule {
   String id;
   List<int> portion = [];
-  List<Timestamp> time = [];
+  List<dynamic> time = [];
```

Comment:
Why change the list do dynamic?

Reply:
This was changed since the Add a Pet UI processes the feeding times as TimeOfDay rather than Timestamp so we switched that to dynamic